docs/questions-and-answers.md

```diff
@@ -3,18 +3,20 @@
 ??? note "What is an autonomous service?"
     An autonomous service is a decentralized service that runs off-chain and provides functionalities to objects living on-chain. Autonomous services are outside the purview and control of a single authority, and can be designed for a variety of purposes, including acting as a decentralized oracle for smart contracts, or executing complex investing strategies that cannot be easily encoded on-chain.
 
-
 ??? note "What is an agent service?"
     An agent service is an autonomous service which is implemented as a multi-agent system using Autonomous Economic Agents (AEAs) using the {{open_autonomy}} framework, which is built on top of {{open_aea}}.
 
-
 ??? note "What is an {{fsm_app}}?"
     An {{fsm_app}} is an application that implements the business logic of an agent service as a finite-state machine. The internal state of an {{fsm_app}} is replicated and synchronized across all the agents forming the agent service.
 
-
 ??? note "What is a keeper agent?"
     It is one of the agents for which the agents have voted to be in charge of executing a certain operation (e.g., settling a transaction on a blockchain).
 
+??? note "What is the consensus gadget?"
+    The consensus gadget is the infrastructure that allows agents in a service to execute a consensus algorithm to synchronize and replicate their internal state. It consists of the consensus gadget nodes (one node per agent) plus the consensus gadget network (which can be the Internet or a dedicated network).
```

Comment:
Consider replacing the first sentence in this para with 

"The consensus gadget is the infrastructure, local to a service, that enables the agents in that service to run a consensus algorithm to synchronize and replicate the service state."

Reply:
fixed